management_api_app/api/routes/workspace_templates.py

```diff
@@ -3,32 +3,81 @@
 from fastapi import APIRouter, Depends, HTTPException
 from starlette import status
 
+from models.domain.resource_template import ResourceTemplate
+from models.schemas.template import TemplateInCreate, TemplateInResponse
+from models.schemas.workspace_service_template import WorkspaceServiceTemplateInCreate, \
+    WorkspaceServiceTemplateInResponse
 from services.authentication import get_current_admin_user
 from api.dependencies.database import get_repository
 from db.errors import EntityDoesNotExist
 from db.repositories.workspace_templates import WorkspaceTemplateRepository
-from models.schemas.workspace_template import (WorkspaceTemplateNamesInList, WorkspaceTemplateInCreate, WorkspaceTemplateInResponse)
+from models.schemas.workspace_template import (WorkspaceTemplateNamesInList, WorkspaceTemplateInCreate,
+                                               WorkspaceTemplateInResponse)
+from models.domain.resource import ResourceType
 from resources import strings
-from services.concatjsonschema import enrich_schema_defs
+from services.concatjsonschema import enrich_schema_defs, enrich_workspace_service_schema_defs
 
 router = APIRouter(dependencies=[Depends(get_current_admin_user)])
 
 
-@router.get("/workspace-templates", response_model=WorkspaceTemplateNamesInList, name=strings.API_GET_WORKSPACE_TEMPLATES)
+@router.get("/workspace-templates", response_model=WorkspaceTemplateNamesInList,
+            name=strings.API_GET_WORKSPACE_TEMPLATES)
 async def get_workspace_templates(
         workspace_template_repo: WorkspaceTemplateRepository = Depends(get_repository(WorkspaceTemplateRepository))
 ) -> WorkspaceTemplateNamesInList:
     workspace_template_names = workspace_template_repo.get_workspace_template_names()
     return WorkspaceTemplateNamesInList(templateNames=workspace_template_names)
 
 
-@router.post("/workspace-templates", status_code=status.HTTP_201_CREATED, response_model=WorkspaceTemplateInResponse, name=strings.API_CREATE_WORKSPACE_TEMPLATES)
+@router.post("/workspace-templates", status_code=status.HTTP_201_CREATED, response_model=WorkspaceTemplateInResponse,
+             name=strings.API_CREATE_WORKSPACE_TEMPLATES)
 async def create_workspace_template(
-    workspace_template_create: WorkspaceTemplateInCreate,
-    workspace_template_repo: WorkspaceTemplateRepository = Depends(get_repository(WorkspaceTemplateRepository)),
+        workspace_template_create: WorkspaceTemplateInCreate,
+        workspace_template_repo: WorkspaceTemplateRepository = Depends(get_repository(WorkspaceTemplateRepository)),
+) -> TemplateInResponse:
+    template_created = create_template_by_resource_type(workspace_template_create, workspace_template_repo,
+                                                        ResourceType.Workspace)
+    template = enrich_schema_defs(template_created)
+    return template
+
+
+@router.post("/workspace-service-templates", status_code=status.HTTP_201_CREATED,
```

Comment:
nit: to make it easier to find, can we split into a separate py file - this is about /workspace-service-templates which is a separate route from /workspace-templates - each route so far has its separate file under routes